docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -451,17 +480,15 @@ The values set under these permission data keys **MUST be of the following forma
 
 :::caution
 
-To **add / remove entries in the list of allowed calls or ERC725Y Data Keys**, the **whole compact bytes array** should be encoded again and reset. Each update **overrides the entire previous state**.
-
-Note that this process is expensive since the data being set is a **CompactBytesArray**.
+To **add / remove entries in the list of allowed calls or ERC725Y Data Keys**, the **whole compact bytes array** should be encoded again and reset. Each update **overrides the entire previous state**. Note that this process can be expensive in gas depending on the total number of allowed calls.
```

Comment:
```suggestion
To **add or remove entries in the list of allowed calls or ERC725Y Data Keys**, the **whole compact bytes array** should be encoded again and reset. Each update **overrides the entire previous state**. Note that this process can be expensive in gas, depending on the total number of allowed calls.
```

If `add / remove` had spaces for clarity, I would suggest using words, as `/` is usually used as a shortcut `add or remove`